src/notifications/generate.ts

```diff
@@ -143,6 +146,15 @@ export const generateNotificationMap: Record<
       .avatarSource(ctx.source)
       .uniqueKey(ctx.bookmark.remindAt.toString())
       .objectPost(ctx.post, ctx.source, ctx.sharedPost),
+  streak_reset_restore: (builder, ctx: NotificationStreakContext) =>
+    builder
+      .icon(NotificationIcon.Streak)
+      .description('Click here if you wish to restore your streak')
+      .uniqueKey(new Date(ctx.streak.lastViewAt).toISOString().split('T')[0])
```

Comment:
let's use `format` from date-fns `dd-MM-YYYY`

Reply:
Done 🥳 

Just used `yyyy` - console was throwing an error saying I should use it over the uppercase version.